virtool/config/cls.py

```diff
@@ -56,6 +56,7 @@ class ServerConfig:
     redis_connection_string: str
     use_b2c: bool
     sentry_dsn: Optional[str]
+    cli_flags: List[str] = field(default_factory=list)
```

Comment:
Redundant prefix. No other options start with `cli_`...
```suggestion
    flags: List[str] = field(default_factory=list)
```